packages/core/src/provider/base.ts

```diff
@@ -482,7 +482,7 @@ export class SmartAccountProvider<
     const maxFeePerGasBid =
       BigInt(feeData.maxFeePerGas) -
       BigInt(feeData.maxPriorityFeePerGas) +
-      maxPriorityFeePerGasBid;
+      maxPriorityFeePerGasBid!;
```

Comment:
nit: can't this be undefined? how does the math work here if undefined? should it be:
```suggestion
      maxPriorityFeePerGasBid ?? 0n;
```

Reply:
I don't think this can be undefined right?